<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PupilDetect</class>
 <widget class="QWidget" name="PupilDetect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2200</width>
    <height>1400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);</string>
  </property>
  <widget class="QLabel" name="displayLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>src</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="LightRecognition">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>1200</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>光斑识别</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>1200</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:20px;</string>
   </property>
   <item>
    <property name="text">
     <string>        设备</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="PupilRecognition">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>1200</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>瞳孔识别</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>1200</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:30px;</string>
   </property>
   <property name="text">
    <string>开始流</string>
   </property>
  </widget>
  <widget class="QLabel" name="displayLabel_2">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>1090</y>
     <width>441</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DarknessPushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>1200</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>最暗点</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_2">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>1280</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:30px;</string>
   </property>
   <property name="text">
    <string>全部开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>1280</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:30px;</string>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>1280</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:30px;</string>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
